packages/odyssey-react-mui/src/labs/SideNav/SideNav.tsx

```diff
@@ -267,17 +375,19 @@ const SideNav = ({
       <SideNavExpandContainer
         odysseyDesignTokens={odysseyDesignTokens}
         isSideNavCollapsed={isSideNavCollapsed}
+        expandedWidth={expandedWidth}
         data-se="expanded-region"
       >
         <Box sx={sideNavHeaderContainerStyles}>
```

Comment:
This `Box` should probably be a `nav`. We'll also need to label it some how I think. Something like this maybe?
```suggestion
        <Box aria-label={navHeaderText} component="nav" sx={sideNavHeaderContainerStyles}>
```